src/docdb/nodes.ts

```diff
@@ -85,8 +85,8 @@ export class DocDBDatabaseNode implements INode {
 
 	get iconPath(): any {
 		return {
-			light: path.join(__filename, '..', '..', '..', '..', 'resources', 'icons', 'light', 'Database.svg'),
-			dark: path.join(__filename, '..', '..', '..', '..', 'resources', 'icons', 'dark', 'Database.svg')
+			light: path.join(__filename, '..', '..', '..', '..', 'resources', 'icons', 'theme-agnostic', 'Azure DocumentDB - database LARGE.svg'),
```

Comment:
Should these files be renamed? 
I've left the names as is from the [public icon set](https://www.microsoft.com/en-us/download/details.aspx?id=41937).

Reply:
I don't have a strong preference - but how do these icons help solve the main issue? I thought the main goal was to get different icons for graph/mongo/table/docdb